### DISCUSSION TITLE
micro_wake_word not work

### DISCUSSION THREAD

**Discussion by devalgo1** [dscripka/openWakeWord#169] [Q&A] [answered]
I'm training a model in automatic_model_training_simple.ipynb for use in micro_wake_word, everything seems to be training well, but when I run the model on the device I get an error:
```
Failed to allocate tensors for the streaming model
[E][micro_wake_word:067]: Failed to initialize models
```

When using standard models everything works.
I noticed several errors in colab:

```
TensorFlow Addons (TFA) has ended development and introduction of new features.
TFA has entered a minimal maintenance and release mode until a planned end of life in May 2024.
Please modify downstream libraries to take dependencies from other repositories in our TensorFlow community (e.g. Keras, Keras-CV, and Keras-NLP).
```

How to overcome this?

**Comment by dscripka** [comment, marked as answer]
Are you referring this repository? https://github.com/kahrendt/microWakeWord

If so, unfortunately the openWakeWord models are not compatible with microWakeWord, as it uses a entirely different (and much smaller) model that is suitable for deployment on embedded devices. If you want a custom microWakeWord model, you would have to train it from scratch using the code in that repo.

**Reply by devalgo1** [reply]
Somehow I managed to miss this))) thanks!!!
